extensions/amp-story-dev-tools/0.1/amp-story-dev-tools.js

```diff
@@ -32,21 +35,169 @@ const fontsToLoad = [
   },
 ];
 
+/**
+ * Generates the template for the root layout.
+ * @param {!Element} element
+ * @return {!Element}
+ */
+const buildContainerTemplate = (element) => {
+  const html = htmlFor(element);
+  return html`
+    <div class="i-amphtml-story-dev-tools-container">
+      <div class="i-amphtml-story-dev-tools-header">
+        <span class="i-amphtml-story-dev-tools-brand">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="30"
+            height="30"
+            viewBox="0 0 30 30"
+            fill="none"
+          >
+            <circle cx="15" cy="15" r="15" fill="white"></circle>
+            <path
+              d="M19.5 9.00015C20.3284 9.00015 21 9.67173 21 10.5002V19.5002C21 20.3286 20.3284 21.0002 19.5 21.0002V9.00015Z"
+              fill="#202125"
+            />
+            <path
+              d="M8.25 9.00015C8.25 8.17173 8.92157 7.50015 9.75 7.50015H16.5C17.3284 7.50015 18 8.17173 18 9.00015V21.0002C18 21.8286 17.3284 22.5002 16.5 22.5002H9.75C8.92157 22.5002 8.25 21.8286 8.25 21.0002V9.00015Z"
+              fill="#202125"
+            />
+            <path
+              d="M22.5 10.5002C23.1213 10.5002 23.625 11.0038 23.625 11.6252V18.3752C23.625 18.9965 23.1213 19.5002 22.5 19.5002V10.5002Z"
+              fill="#202125"
+            />
```

Comment:
We can save some bytes with this path like this:
`<path fill="#202125" d="M19.5 9a1.5 1.5 0 011.5 1.5v9a1.5 1.5 0 01-1.5 1.5V9zM8.25 9a1.5 1.5 0 011.5-1.5h6.75A1.5 1.5 0 0118 9v12a1.5 1.5 0 01-1.5 1.5H9.75a1.5 1.5 0 01-1.5-1.5V9zM22.5 10.5c.621 0 1.125.504 1.125 1.125v6.75c0 .622-.504 1.125-1.125 1.125v-9z"/>`

[svgomg](https://jakearchibald.github.io/svgomg/) is a helpful tool for simplifying SVG paths but it's good to check the output. You can see in this case that it's putting the exact same path code in one path, so we can be sure the shapes are the same, it's just less elements.

Reply:
Wow that's really cool. Works as expected